.github/workflows/changelog.yml

```diff
@@ -0,0 +1,31 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Changelog
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  pull_request:
+    branches: [ develop ]
```

Comment:
Might make sense to run it on all PRs, not just into develop?

Reply:
Might want to make sure that doesn't block releases/hotfixes etc. which don't necessarily add any new changes.